### PR TITLE
Gabra Mikāʾel

### PR DESCRIPTION
**Pull request by thea-m**
I now subsumed all DS Gabra Mikāʾels under one ID. What do you think?

### CODE REVIEW

**Comment by thea-m** [comment]
There is actually the Walatta Māryām (PRS12967WalattaM) whose name as been erased and over which Gabra Mikāʾel has been written in DSEtiop9. But it is all very speculative.

File: PRS12001-13000/PRS12002Gabrami.xml [hunk 1]
```diff
@@ -40,17 +40,43 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                        <placeName ref="INS0352EA"/> in the manuscripts attesting him.</residence>
+                    <floruit notBefore="1875" notAfter="1960"/>
+                    <note>
+                        <p>The modern Arabic manuscript description placed on a paper sheet inside <ref type="mss" corresp="DSEthiop13"/> states
+                        that Mīḫāʾīl al-Ḥabašī was born in <placeName ref="LOC1178AddiNa"/> in Eritrea and that his mother's name was
+                        <foreign xml:lang="ar">Lītā Māryām</foreign>, possibly a transcription of <foreign xml:lang="gez">Walatta Māryām</foreign>.</p>
+                        
+                      <p>  The <ref target="https://stmaryrodelfarag.com/%D8%A7%D8%A8%D8%A7%D8%A1-%D8%B3%D8%A7%D8%A8%D9%82%D9%8A%D9%86/">website of the 
+                            church St Mary in Rawḍ al-Farag in Cairo</ref> states that a <foreign xml:lang="ar">qummuṣ</foreign> Mīḫāʾīl al-Ḥabašī lived in this church, until he was ordered by
+                        <persName ref="PRS3260CyrilVI"/> to return to a monastery. Since it was impossible for him to return to Ethiopia, he went to <placeName ref="INS0478DS"/>,
```
Comment (DenisNosnitsin1970) [inline PRS12001-13000/PRS12002Gabrami.xml:68]: to a monastery - Dayr as-Suryan or his Ethiopian monastery? (then better to "his monastery in his native country"). "To return to Eritrea", then, or just "to return"

Comment (DenisNosnitsin1970) [reply PRS12001-13000/PRS12002Gabrami.xml:68]: To erase his mother's name - I think it is possible, for instance if he inherited the book or she just gave it to as present him.

Comment (thea-m) [reply PRS12001-13000/PRS12002Gabrami.xml]: Ah yes, I misunderstood the Arabic. It was apparently a general order (I'm trying to find a date now), not specific to this monk.

**Comment by thea-m** [comment]
@DenisNosnitsin1970

**Pull request merged**